### PR TITLE
More pages2

### PR DESCRIPTION
**Pull request by Oasixer**
### What are you trying to accomplish?
Add not_implemented stub pages and linked to them from topbar links and register link

### Is there anything reviewers should know?
no

### If needed, did you change documentation(`README.md`, `ARCHITECTURE.md`, etc) to reflect your changes?
ye
- [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

File: frontend/src/components/general/scss/fonts.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import url(http://fonts.googleapis.com/css?family=Roboto+Slab|Open+Sans|Poppins:400italic,700italic,300,400,700);
```
Comment (prairir) [inline frontend/src/components/general/scss/fonts.scss:1]: i dont understand why you would do this and add the fonts within

Comment (Oasixer) [reply frontend/src/components/general/scss/fonts.scss:1]: wym? its just one of 1 trillion ways of importing google fonts. can you put a snippet of your preferred way?

Comment (prairir) [reply frontend/src/components/general/scss/fonts.scss:1]: you do that but also add the files of fonts like `frontend/public/assets/fonts/Poppins/Poppins-BlackItalic.ttf`. i dont understand why both

Comment (Oasixer) [reply frontend/src/components/general/scss/fonts.scss:1]: oh oops, thanks. I tried to get local assets working then did the @import instead. ill delete the assets

**Pull request merged**